tools/dev_qa.sh

```diff
@@ -79,13 +94,15 @@ tensorflow_iid=$(invocation_id)
 run_test \
     https://github.com/buildbuddy-io/buildbuddy \
     buildbuddy \
+    3 \
```

Comment:
Since this is an "optional" param you could alternatively set it as a variable scoped to the run_test command, like

`NUM_RETRIES=3 run_test ...`

Then in the func

```
: "${NUM_RETRIES:=0}"
```

Reply:
> Is this flaky because of a certain test, or are we failing to fetch some deps or something like that? (If it's the former, bazel lets you set the retry count per-test; see `--flaky_test_attempts` in our .bazelrc for an example)

Thanks that's simpler. Flaky because of certain tests. Made a new PR https://github.com/buildbuddy-io/buildbuddy/pull/3398/files